src/emails/EmailController.ts

```diff
@@ -0,0 +1,23 @@
+import { Router, Request, Response } from 'express';
+import { EmailService } from './EmailService';
+
+// Dependencies
+const emailSrv: EmailService = new EmailService();
+
+// Assign router to the express.Router() instance
+const emailRouter: Router = Router();
+
+/**
+ * Method: 
+ * Verb: POST
+ * Route: /api/events
+ */
+emailRouter.post('/', async(req: Request, res: Response) => {
+	let { from, to, subject, message } = req.body;
```

Comment:
- Message parameter should be sanitized.
- Best practice: body message parameterized is stored in the server (mongo collection).